web/libs/core/src/pages/AccountSettings/AccountSettings.tsx

```diff
@@ -0,0 +1,72 @@
+import { Card, CardHeader, CardTitle, CardContent, CardDescription } from "@humansignal/ui/lib/card-new/card";
```

Comment:
Can we clean up the imports to being something a bit more concise than the full folder path? Doesn't have to be right now, I just think going forward if the paths are all long it appears messy similar to relative imports.

Reply:
suggestions? the only thing i see is to add `@humansignal/components` alias that points directly to the `lib/` folder